Inferences/InductionHelper.cpp

```diff
@@ -139,15 +140,13 @@ bool InductionHelper::isIntInductionOn() {
 
 bool InductionHelper::isIntInductionOneOn() {
   CALL("InductionHelper::isIntInductionOneOn");
-  static bool one = env.options->intInduction() == Options::IntInductionKind::ONE ||
-                    env.options->intInduction() == Options::IntInductionKind::ALL;
+  static bool one = env.options->intInduction() == Options::IntInductionKind::ONE;
```

Comment:
Could this just be a non-static `bool` in these cases since we're changing it? The corresponding load/eq instructions are likely to be faster as statics are surprisingly expensive.

Reply:
Thanks, pet peeve. :-) It's not a ridiculous overhead so they definitely have their place, but I think it's more than people expect given that it's a keyword. Post C++11 it has to be thread-safe as well. Have a play with https://godbolt.org/z/feTzv4oE3 if you're interested.